Sources/Wrappers/AngouriMath.FSharp/Functions.fs

```diff
@@ -273,45 +264,4 @@ let solutions x expr =
 type Lengths =
```

Comment:
If I recall correctly this thing is only used by matrices (to track their size), so I guess can be moved to them too